admin_manual/installation/nginx_configuration.rst

```diff
@@ -14,7 +14,9 @@ Nginx Configuration
 -  ``add_header`` statements are only taken from the current level and are not cascaded 
    from or to a different level. All necessary ``add_header`` statements must be defined 
    in each level needed. For better readability is is possible to move *common* add 
```

Comment:
While you are here, this typo could be fixed also.
s/is is possible/it is possible/